platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/util/function/EBiConsumer.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.util.function;
```

Comment:
❗️ Since this and `ERunnable` are part of the `platform-migratable-api` bundle, and to avoid split packages down the road (see https://www.ibm.com/developerworks/websphere/techjournal/1007_charters/1007_charters.html#sec8), we should either move them to their own API bundle or keep them under the same package as the rest of the platform API interfaces.

Reply:
@lessarderic - Not a fan of changing the package since these are utility classes that could be used elsewhere. So which bundle should it move to? Do we have one for common utility classes like that?

Reply:
We currently don't have anything for this type of things, so we will need to create one. How about a new module under `libs` called `common`, `util` or `core` maybe?

Reply:
Opened DDF-3275 for that